python/src/robyn/robyn.py

```diff
@@ -398,29 +400,49 @@ def optimize_budget(
             logger.error("Budget optimization failed: %s", str(e))
             raise
 
-    @staticmethod
-    def generate_one_pager(
-        pareto_result: ParetoResult,
-        cluster_result: Optional[ClusteringConfig] = None,
-        mmm_data: Optional[MMMData] = None,
-        plots: Optional[List[str]] = None,
+    def plot_one_pager(self, solution_id: Optional[str] = None) -> None:
```

Comment:
This wrapper method on top of _generate_one_pager doesn't do anything. I would recommend just make generate_one_pager method public. Also, let's plot all plots from this method. No need to keep plotType input parameter here in this interface.